web/src/lib/domain-logic/essentialQuestions.ts

```diff
@@ -108,7 +108,7 @@ export const EssentialQuestions: EssentialQuestion[] = [
   new EssentialQuestion({
     shouldBeResetWhenIndustryChanges: true,
     isQuestionApplicableToIndustry: (industry) => {
-      return !!industry.industryOnboardingQuestions.isChildcareForSixOrMore && industry.id === "daycare";
+      return !!industry.industryOnboardingQuestions.isChildcareForSixOrMore;
     },
     fieldName: "isChildcareForSixOrMore",
   }),
```

Comment:
Beyond the "scope" of this chore, but given that all of these `EssentialQuestions` are generated in a _very_ similar, maybe there's another refactor opportunity here.